lucet-wasi/build.rs

```diff
@@ -32,6 +34,10 @@ fn wasm_clang_root() -> PathBuf {
     }
 }
 
+#[cfg(not(feature = "bindgen"))]
```

Comment:
Can you add a comment explaining why this exists?

Reply:
Done.